pymeasure/display/windows/managed_window.py

```diff
@@ -630,7 +537,8 @@ def directory(self, value):
 
 class ManagedWindow(ManagedWindowBase):
```

Comment:
Just a thought; would it not also make sense to split this file into one for ManagedWindowsBase and one for ManagedWindow?

Reply:
I'd say multiple classes per module is definitely fine in python, and imo having every class in a separate module does not improve the code structure.